domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/PartAdmissionResponse.java

```diff
@@ -0,0 +1,135 @@
+package uk.gov.hmcts.cmc.domain.models;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.google.common.collect.ImmutableList;
+import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
+import org.hibernate.validator.constraints.NotBlank;
+import uk.gov.hmcts.cmc.domain.models.legalrep.StatementOfTruth;
+import uk.gov.hmcts.cmc.domain.models.party.Party;
+import uk.gov.hmcts.cmc.domain.models.response.DefendantPaymentPlan;
+import uk.gov.hmcts.cmc.domain.models.response.EvidenceItem;
+import uk.gov.hmcts.cmc.domain.models.response.HowMuchOwed;
+import uk.gov.hmcts.cmc.domain.models.response.TimelineEvent;
+import java.util.Objects;
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+import static uk.gov.hmcts.cmc.domain.utils.ToStringStyle.ourStyle;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Please remove if possible, we should only send properties which are understandable by backend, this might hide some serialisation problem. I prefer to fail fast.